### DISCUSSION TITLE
VM Information including tag categories and the tag values.

### DISCUSSION THREAD

**Discussion by Steve-Groner** [vmware/govmomi#3212] [Community Help]
I am trying to export all VM information from a cluster and I was hoping I could include all the tag categories and values.  If I cannot do them in one pass, how would I get the tag cetegories.  the tags.attached.ls -r {vm} will give me the tag values but it doesnt give me the category names.  Any help would be appreciated.  

BTW, I am using the govc ls -json to get the information for the VM's, I just need to get the tag cats and values to be able to complete what I am doing.

**Comment by dougm** [comment]
Looks we'd need to add a flag to `tags.attached.ls` that would also output the category. You might be able to get by using `govc tags.ls` to lookup the category?

**Reply by tenthirtyam** [reply]
@Steve-Groner - did `govc tags.ls` work for you?
